src/components/Trends/Explore/TrendsExploreAdditionalInfo.js

```diff
@@ -0,0 +1,120 @@
+import React, { useState } from 'react'
+import { compose } from 'recompose'
+import { graphql } from 'react-apollo'
+import { Tabs } from '@santiment-network/ui'
+import { DAY, getTimeIntervalFromToday } from '../../../utils/dates'
+import { ALL_INSIGHTS_BY_TAG_QUERY } from '../../../queries/InsightsGQL'
+import { NEWS_QUERY } from '../../News/NewsGQL'
+import {
+  getInsightTrendTagByDate,
+  oneDayTimestamp
+} from '../../Insight/InsightsTrends'
+import InsightCard from '../../Insight/InsightCard'
+import News from '../../News/News'
+import styles from './TrendsExploreAdditionalInfo.module.scss'
+
+const NEWS_INDEX = 'News'
+const INSIGHTS_INDEX = 'Insights'
+
+const TrendsExploreAdditionalInfo = ({ news, insights }) => {
+  let [selectedTab, setSelectedTab] = useState(null)
+
+  function handleSelectTab (tab) {
+    if (tab !== (selectedTab || defaultSelectedTab)) {
+      setSelectedTab(tab)
+    }
+  }
+
+  const newsTab = {
+    index: NEWS_INDEX,
+    content: `${NEWS_INDEX} (${news.length})`
+  }
+
+  const insightsTab = {
+    index: INSIGHTS_INDEX,
+    content: `${INSIGHTS_INDEX} (${insights.length})`
+  }
+
+  const tabs = []
+  if (insights.length > 0) tabs.push(insightsTab)
+  if (news.length > 0) tabs.push(newsTab)
+
+  let defaultSelectedTab
+
+  if (tabs.length === 0) return null
+  else defaultSelectedTab = tabs[0].index
```

Comment:
@haritonasty Do we really need "else" here?
If tabs.length === 0 ok. return null.
After const defaultSelectedTab = tabs[0].index